compiler/debug.cpp

```diff
@@ -57,6 +57,7 @@ std::string debugTokenName(TokenType t) {
     {tok_as, "tok_as"},
     {tok_case, "tok_case"},
     {tok_switch, "tok_switch"},
+    {tok_match, "tok_match"},
```

Comment:
Add a lexer test case for a new token?

Reply:
I haven't started making tests yet, this is an early draft.

Reply:
Added some tests for lexer.